ap-score-hider.user.js

```diff
@@ -1,8 +1,8 @@
 // ==UserScript==
-// @name         AP® Score Hider
+// @name         AP® Score Hider and Reorder
 // @namespace    http://tampermonkey.net/
-// @version      1.2.4
-// @description  Hides AP® Exam scores on the College Board website until you click on them. As a bonus, display confetti when you click on a passing exam.
+// @version      1.2.5
```

Comment:
per semver (semantic versioning), this kind of change would actually be a minor change, not a bugfix, so this would be 1.3.0 https://semver.org/